File: CS/CodeBehind/EntityFramework/InstantFeedbackMode/EditIssueInfo.cs
using DevExpress.Mvvm;
using System.Collections;
using EntityFrameworkIssues.Issues;

namespace EntityFrameworkIssues {
    public class EditIssueInfo : BindableBase {
        public EditIssueInfo(IssuesContext dbContext, IList users) {
            DbContext = dbContext;
            Users = users;
        }
        public IssuesContext DbContext { get; }
        public IList Users { get; }
    }
}

File: VB/CodeBehind/EntityFramework/InstantFeedbackMode/EditIssueInfo.vb
Imports DevExpress.Mvvm
Imports System.Collections.Generic
Imports EntityFrameworkIssues.Issues

Public Class EditIssueInfo
    Inherits BindableBase

    Public Sub New(ByVal dbContext As IssuesContext, ByVal users As IList)
        Me.DbContext = dbContext
        Me.Users = users
    End Sub

    Public ReadOnly Property DbContext As IssuesContext
    Public ReadOnly Property Users As IList
End Class
